### ISSUE TITLE
Error in CommitsLOC extension

### ISSUE THREAD

**Issue by rodrigokuroda** [MetricsGrimoire/CVSAnalY#94]
This error is occurring during extensions execution, specifically the CommitsLOC.

```
$ cvsanaly2 --debug --writable-path=aries --save-logfile=aries/logfile --db-user=USER --db-password=PASS 
  --db-database=aries --metrics-all --extensions=CommitsLOC,CommitsLOCDet,FileTypes,Months,Patches,Weeks 
  https://svn.apache.org/repos/asf/aries/
DBG: INSERT INTO repositories (id, uri, name, type) values (%s, %s, %s, %s)
Parsing log for https://svn.apache.org/repos/asf/aries (svn)
['svn', '-v', 'log', 'https://svn.apache.org/repos/asf/aries']
DBG: SVN Parser: File /aries/trunk/subsystem/subsystem-core has been copied to /aries/tags/org.apache.aries.subsystem.core-1.1.0
DBG: SVN Parser: File /aries/trunk/subsystem/subsystem-core has been copied to /aries/tags/org.apache.aries.subsystem.core-1.0.2

[... omitted logs ...]

Executing extensions
Executing extension Patches
DBG: SELECT id from repositories where uri = %s
Executing extension Months
Executing extension CommitsLOC
DBG: SELECT id from repositories where uri = %s
DBG: SELECT id, rev, composed_rev from scmlog where repository_id = %s
['svn', 'diff', '-r', '817816:817817', 'https://svn.apache.org/repos/asf']
DBG: INSERT INTO commits_lines (id, commit_id, added, removed) values (%s, %s, %s, %s)
['svn', 'diff', '-r', '819516:819517', 'https://svn.apache.org/repos/asf']
DBG: INSERT INTO commits_lines (id, commit_id, added, removed) values (%s, %s, %s, %s)
['svn', 'diff', '-r', '819894:819895', 'https://svn.apache.org/repos/asf']
DBG: INSERT INTO commits_lines (id, commit_id, added, removed) values (%s, %s, %s, %s)
['svn', 'diff', '-r', '820086:820087', 'https://svn.apache.org/repos/asf']
DBG: INSERT INTO commits_lines (id, commit_id, added, removed) values (%s, %s, %s, %s)
['svn', 'diff', '-r', '820169:820170', 'https://svn.apache.org/repos/asf']
DBG: INSERT INTO commits_lines (id, commit_id, added, removed) values (%s, %s, %s, %s)
['svn', 'diff', '-r', '820226:820227', 'https://svn.apache.org/repos/asf']
DBG: INSERT INTO commits_lines (id, commit_id, added, removed) values (%s, %s, %s, %s)
['svn', 'diff', '-r', '820270:820271', 'https://svn.apache.org/repos/asf']

Error running extension Patches: 'Cursor' object has no attribute 'execut'
```

But the svn diff command is successfully executed.

```
svn diff -r 820270:820271 https://svn.apache.org/repos/asf
[... omitted a very large output  ...]
```

I note that the svn diff output is large and it may be causing this error.

After the error occurring, I try to CRTL+C to exit from process, but another possible error happens:

```
^CTraceback (most recent call last):
  File "/usr/local/bin/cvsanaly2", line 5, in <module>
    pkg_resources.run_script('cvsanaly2==2.1.0', 'cvsanaly2')
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 528, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 1394, in run_script
    execfile(script_filename, namespace, namespace)
  File "/usr/local/lib/python2.7/dist-packages/cvsanaly2-2.1.0-py2.7.egg/EGG-INFO/scripts/cvsanaly2", line 37, in <module>
    retval = pycvsanaly2.main.main (sys.argv[1:])
  File "/usr/local/lib/python2.7/dist-packages/cvsanaly2-2.1.0-py2.7.egg/pycvsanaly2/main.py", line 382, in main
    emg.run_extensions(repo, path or uri, db)
  File "/usr/local/lib/python2.7/dist-packages/cvsanaly2-2.1.0-py2.7.egg/pycvsanaly2/ExtensionsManager.py", line 105, in run_extensions
    self.run_extension(name, extension, repo, uri, db)
  File "/usr/local/lib/python2.7/dist-packages/cvsanaly2-2.1.0-py2.7.egg/pycvsanaly2/ExtensionsManager.py", line 66, in run_extension
    extension.run(repo, uri, db)
  File "/usr/local/lib/python2.7/dist-packages/cvsanaly2-2.1.0-py2.7.egg/pycvsanaly2/extensions/CommitsLOC.py", line 305, in run
    (added, removed) = counter.get_lines_for_revision(revision)
  File "/usr/local/lib/python2.7/dist-packages/cvsanaly2-2.1.0-py2.7.egg/pycvsanaly2/extensions/CommitsLOC.py", line 115, in get_lines_for_revision
    printerr("Error running svn diff command: %s", (str(e)))
  File "/usr/local/lib/python2.7/dist-packages/cvsanaly2-2.1.0-py2.7.egg/pycvsanaly2/utils.py", line 111, in printerr
    str = str % tuple(to_utf8(arg) for arg in args)
TypeError: not all arguments converted during string formatting
```

**Comment by rodrigokuroda** [comment]
I think that I found the cause of this error.

In the line 70 of pycvsanaly2/extensions/Patches.py, instead:

```
cursor.execut("CREATE TABLE patches (" +
```

should be:

```
cursor.execute("CREATE TABLE patches (" +
```

**Comment by andygrunwald** [comment]
I merged PR #95, so this bug is fixed and can be closed as well, right?
If not, please reopen. Thanks!
